src/Common/src/Interop/Windows/mincore/Interop.UnsafeCreateFile.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using Microsoft.Win32.SafeHandles;
+
+internal partial class Interop
+{
+    internal partial class mincore
+    {
+        [System.Security.SecurityCritical]  // auto-generated
+        internal static SafeFileHandle UnsafeCreateFile(
```

Comment:
@JeremyKuhne, regarding long paths, I found some useful implementations in this library: https://github.com/peteraritchie/LongPath (`Pri.LongPath` package @NuGet).